frame.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+# Author: Ruslan Krenzler.
+# Date: 11 February 2018
+# Create a pipe frame box.
+
+import math
+import csv
+import os.path
+
+import FreeCAD
+import Part
+import Draft
+
+import OSEBase
+from piping import *
+import outerCorner
+import pipe as pipemod
+
+tu = FreeCAD.Units.parseQuantity
```

Comment:
Why is this variable called tu? Would parse_quanity or pq be a better name?

Reply:
tu = To Units. But I you are right. I will change it to a better name. Just parseQuantity.